internal/provider/virtual_cluster_resource.go

```diff
@@ -407,17 +417,6 @@ func (r *virtualClusterResource) Delete(ctx context.Context, req resource.Delete
 func (r *virtualClusterResource) ImportState(ctx context.Context, req resource.ImportStateRequest, resp *resource.ImportStateResponse) {
 	// Retrieve import ID and save to id attribute
 	resource.ImportStatePassthroughID(ctx, path.Root("id"), req, resp)
-
```

Comment:
according to the documentation: https://github.com/hashicorp/terraform/blob/main/docs/resource-instance-change-lifecycle.md#importresourcestate

> Terraform Core will always pass the returned Import Stub State to the normal ReadResource operation after ImportResourceState returns it, so in practice the provider may populate only the minimal subset of attributes that ReadResource will need to do its work, letting the normal function deal with populating the rest of the data to match what is currently set in the remote system.

I think we should only perform  `resource.ImportStatePassthroughID` and make sure that `Read` is fetching the latest value for all the attributes